feat(docs/ulule-api): change main_image fields to i18n objects (#89)
ulule/developers.ulule.com: content/docs/ulule-api/reference/project.md

| `is_online`                  | bool                                                        | True if the project is online                                                                                                                                                  |
| `lang`                       | string                                                      | Main language of the project                                                                                                                                                   |
| `location`                   | [location resource](#location-resource)                     | Location of the project                                                                                                                                                        |
| `lowest_contribution_amount` | int                                                         | Lowest contribution possible in the project currency                                                                                                                           |

> | `main_image`                 | [i18n object](#i18n) with [image resource](#image) as value | Main image of the project                                                                                                                                                      |

Reviewer: ```suggestion
| `main_image`                 | [i18n object](#i18n) where values are [image resources](#image) | Main image of the project                                                                                                                                                      |
```
